src/vocab/subword.rs

```diff
@@ -144,21 +147,63 @@ where
 {
     fn from(builder: VocabBuilder<SubwordVocabConfig<BucketConfig>, T>) -> Self {
         let config = builder.config;
+        let vocab_cutoff = config.vocab_cutoff;
+        match vocab_cutoff {
+            VocabCutoff::MinCount(min_count) => {
```

Comment:
There is a lot of overlap with `SimpleVocab::from`. It should be possible to factor out the both versions of filtering to a generic function that is used by both `from` implementations?

Reply:
>`SimpleVocab` is also used for dependency embeddings, in that case the dependency contexts are stored in a `SimpleVocab`.

Ah thanks!

>  We didn't catch that when implementing the no-subwords training but in light of #79 I think it's not really an issue that needs to be fixed.

If the ```EOS``` is going to be kicked out anyway, I think it would be safe if I just remove it here :)

Thanks!